api/app/signals/apps/email_integrations/actions/abstract.py

```diff
@@ -88,7 +88,7 @@ def render_mail_data(self, context):
 
         return subject, message, html_message
 
-    def send_mail(self, signal, dry_run=False):
+    def send_mail(self, signal, recipient_list=[], dry_run=False):
```

Comment:
It is better to override the `send_mail` in the `AssignedAction` to fit this solution instead of trying to make it more generic. This will also keep things more readable and when looking at the `AssignedAction` it is clear that this is an action that will not mail reporters but "ambtenaren".

Reply:
Implemented this 👍